### PR TITLE
IO1-3038: changed font size of references match the other text

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/components/applications/SSV/MyAccount/common/LiquidationStateError/LiquidationStateError.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const LiquidationStateError = (props: Props) => {
 
     const renderText = () => {
         if (errorType === ErrorType.WithdrawAll) {
-            return <div>Withdrawing the requested amount will liquidate your cluster, which will result in inactivation (<LinkText text={'penalties on the beacon chain'} link={'https://launchpad.ethereum.org/en/faq#responsibilities'} />) of your validators, as they will no longer be operated by the network.</div>;
+            return <div>Withdrawing the requested amount will liquidate your cluster, which will result in inactivation (<LinkText className={classes.LinkText} text={'penalties on the beacon chain'} link={'https://launchpad.ethereum.org/en/faq#responsibilities'} />) of your validators, as they will no longer be operated by the network.</div>;
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/common/LiquidationStateError/LiquidationStateError.tsx:29]: Move the link to config constants

File: src/app/components/applications/SSV/MyAccount/common/LiquidationStateError/LiquidationStateError.tsx [hunk 2]
```diff
@@ -62,8 +62,8 @@ const LiquidationStateError = (props: Props) => {
           {renderText()}
         </Grid>
         {errorType !== 4 && errorType !== 5 && (
-          <Grid item className={classes.LinkText}>
-            <LinkText text={'Read more on liquidations'} link={'https://docs.ssv.network/learn/protocol-overview/tokenomics/liquidations'} />
+          <Grid item>
+            <LinkText className={classes.LinkText} text={'Read more on liquidations'} link={'https://docs.ssv.network/learn/protocol-overview/tokenomics/liquidations'} />
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/MyAccount/common/LiquidationStateError/LiquidationStateError.tsx:66]: Link to config

**Review by meshin-blox — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/MyAccount/common/LiquidationStateError/LiquidationStateError.tsx (hunk 1 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/common/LiquidationStateError/LiquidationStateError.tsx]: done

File: src/app/components/applications/SSV/MyAccount/common/LiquidationStateError/LiquidationStateError.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/common/LiquidationStateError/LiquidationStateError.tsx]: done

**Review by meshin-blox — APPROVED** [review]

**Pull request merged**